src/defs.hpp.in

```diff
@@ -45,8 +45,20 @@
 // include self gravity? default=0 (false)
 #define SELF_GRAVITY_ENABLED @SELF_GRAVITY_ENABLED@
 
+// include chemistry? default=0 (false)
+#define CHEMISTRY_ENABLED @CHEMISTRY_ENABLED@
+
+// ChemNetwork class header file included in species.hpp
+#define CHEMNETWORK_HEADER "@CHEMNETWORK_HEADER@"
+
+// CVODE solver option (CVODE or NO_CVODE)
+#define @CVODE_OPTION@
```

Comment:
Might want to change this to be similar to the other switches, i.e. 0 or 1

Reply:
The `#ifdef CVODE` macro is used to deal with the external CVODE library (see `src/chemistry/network_wrapper.cpp(and .hpp)`). So I think the 0 or 1 switch doesn't make sense?

Reply:
Oh, I see what your saying. In this case it should be moved down to the following section so it is grouped with the similarly-formatted FFT, HDF5, etc. flags